### PR TITLE
Include broker name in `cf services` output

### PR DESCRIPTION
**Pull request by nmaslarski**
When executing `cf services` command you get the list of services in your org/space.
This PR adds a column showing the service broker name of these services.

More information on https://www.pivotaltracker.com/n/projects/2105761/stories/160752090

Best,
Niki
On behalf of SAPI Team

### CODE REVIEW

File: integration/shared/isolated/services_command_test.go [hunk 1]
```diff
@@ -123,9 +123,9 @@ var _ = Describe("services command", func() {
 		It("displays all service information", func() {
 			session := helpers.CF("services")
 			Eventually(session).Should(Say("Getting services in org %s / space %s as %s...", orgName, spaceName, userName))
-			Eventually(session).Should(Say(`name\s+service\s+plan\s+bound apps\s+last operation`))
-			Eventually(session).Should(Say(`%s\s+%s\s+%s\s+%s\s+%s`, managedService1, service, servicePlan, appName1, "create succeeded"))
-			Eventually(session).Should(Say(`%s\s+%s\s+%s\s+%s, %s\s+%s`, managedService2, service, servicePlan, appName1, appName2, "create succeeded"))
+			Eventually(session).Should(Say(`name\s+service\s+plan\s+bound apps\s+last operation\s+broker`))
```
Comment (willmurphyscode) [inline integration/shared/isolated/services_command_test.go]: These tests will fail when run against our legacy environment. Please conditionally skip them based on the CC API version when `service_broker_name` was added to the JSON payload.

The current behavior is that when the service broker name is unavailable, the column is displayed but its value is always empty. Please also add a test that only runs on the legacy environment that asserts that the command succeeds and displays the empty column.

Comment (willmurphyscode) [reply integration/shared/isolated/services_command_test.go]: Please add a constant for the lowest version where the broker name will show up here: https://github.com/cloudfoundry/cli/blob/71ba30a7fb3bb3177b49dd6f82c943db55ea2686/api/cloudcontroller/ccversion/minimum_version.go#L3

Comment (ablease) [reply integration/shared/isolated/services_command_test.go]: Hey @willmurphyscode. We have updated this PR so that we conditionally skip these tests when the CC API version is lower than `ccversion.MinVersionServiceBrokerNameV2`.  Please see this [commit](https://github.com/cloudfoundry/cli/pull/1538/commits/7785447f585923697a128cd8b22e698cf3070c56)

Thanks!

Alex + @nmaslarski

Comment (tjvman) [reply integration/shared/isolated/services_command_test.go]: These tests are now working against my local legacy environment, so I'll resolve this conversation.
